components/Carousel.vue

```diff
@@ -0,0 +1,188 @@
+    sliders: {
+      type:    Array,
+      default: null
+    }
+  },
+  data() {
+    return {
+      slider:          this.sliders,
+      activeItemId:    0
+    };
+  },
+
+  computed: {
+
+    trackStyle() {
+      return `transform: translateX(-${ this.activeItemId * 100 / this.slider.length }%); width: calc(${ 60 * this.slider.length }%)`;
+    },
+
+  },
+
+  methods: {
+    scrollSlide(i) {
+      this.activeItemId = i;
+      setTimeout(() => {
+        if (i <= 1) {
+          this.$refs.slide[this.slider.length - 1].style.left = '-93%';
+          this.$refs.slide[0].style.left = '7%';
+        } else {
+          this.$refs.slide[this.slider.length - 1].style.left = '7%';
+          this.$refs.slide[0].style.left = '107%';
+        }
+      }, 400);
+    },
+  }
+};
+
+</script>
+
+<template>
+  <div class="slider">
+    <div id="slide-track" ref="slider" style="transition: 1s ease-in-out" :style="trackStyle" class="slide-track">
+      <div
+        v-for="(slide, i) in slider"
+        :id="`slide` + i"
+        ref="slide"
+        :key="i"
+        class="slide"
+      >
+        <div class="slide-header">
+          Featured chart
+        </div>
+        <div class="slide-content">
+          <div class="slide-img">
+            <img :src="require(`~/assets/images/featured/${slide.img}`)" />
+          </div>
+          <div class="slide-content-right">
+            <BadgeState label="Partner" color="slider-badge  mb-20" />
+            <h1>{{ slide.title }}</h1>
+            <p>{{ slide.content }}</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="controls">
+      <div
+        v-for="(slide, i) in slider"
+        :key="i"
+        class="control-item"
+        :class="{'active': activeItemId === i}"
+        @click="scrollSlide(i, slider.length)"
+      ></div>
+    </div>
+  </div>
+</template>
+
+<style lang='scss'>
+ .slider {
+  margin: auto;
+  position: relative;
+  width: 100%;
+  place-items: center;
+  overflow: hidden;
+  margin-bottom: 30px;
+  min-width: 700px;
+}
+
+.slide-track {
+  display: flex;
+  // width: calc(60% * 5);
+  animation: scrolls 10s ;
+  position: relative;
+}
+
+.slider-badge {
+  background: var(--app-partner-accent);
+  color: var(--body-bg);
+}
+.slide {
+  min-height: 210px;
+  width: 60%;
+  max-width: 60%;
+  margin: 0 10px;
+  position: relative;
+  border: 1px solid var(--tabbed-border);
+  border-radius: var(--border-radius);
+  left: 7%;
+
+  &:last-child {
+    left: -93%;
+  }
+
+  .slide-header {
+    background: var(--default);
+    width: 100%;
+    padding: 10px 15px;
+  }
+  .slide-content {
+    display: flex;
+    padding: 30px;
+
+    .slide-img {
+      width: 150px;
+
+      img {
+        width: 100%;
+      }
+    }
+
+    .slide-content-right {
+      border-left: 1px solid var(--tabbed-border);
+      margin-left: 30px;
+      padding-left: 30px;
+
+      span {
+        margin: 0;
+      }
+
+    }
+  }
+
+}
+
+.slider::before,
```

Comment:
Please replace these before/after selector with actual elements, so it is possible to navigate also by clicking at the left/right area. Currently is really not friendly the navigation.

Reply:
I feel this fed-out effect is just for visual presentation. I think we should keep it simple as per the design.

Reply:
I am personally indifferent to the choice, but it was really difficult to navigate through. Maybe check this out with @kwwii to be sure?

Reply:
Fixed